components/bl-webcamPhoto-component/src/popup/options.js

```diff
@@ -0,0 +1,11 @@
+export const PopupOptionsMap = {
```

Comment:
This and popup/index.js could be combined into one file moved to src/popup.js

Also, add please screenshots for popups

Reply:
![image](https://user-images.githubusercontent.com/88846606/178752647-75a61e83-de45-486f-9917-798214e72a9f.png)